ha_cluster_exporter.go

```diff
@@ -149,50 +165,60 @@ func main() {
 	// for each different metrics, handle it in differents gorutines, and use same timeout.
 
 	// set DRBD metrics
-
 	go func() {
 		log.Infoln("Starting DRBD metrics collector...")
-
-		started := false
-
 		for {
-			if started {
-				time.Sleep(time.Duration(int64(*timeoutSeconds)) * time.Second)
-			} else {
-				started = true
-			}
```

Comment:
This was here to avoid sleeping on the first iteration, since users can set the timeout arbitrarily. You didn't like it? :crying_cat_face:

Reply:
@stefanotorresi  nope. Imho we are complictating the thing. for the 1st iteration, most of the user will even not notice that it sleeps 5 sec

Reply:
I see your point, OTOH as a user I could set the period to 30 seconds, and nothing would happen until that time passes at least once, and to be honest I wouldn't like this UX at all...

Reply:
I'm sorry, man, but we'll have to agree to disagree on this one...  :stuck_out_tongue_winking_eye: 

If you proposed a different, more elegant solution, I would be totally fine with it, but rather than just removing the functionality, I think we should leave it as it is, instead of knowingly introducing a bug for the sake of subjective readability.

Maybe we should ask for another opinion?

Reply:
I don't introduce any bug , is just a different behavior. Your example with 30 Seconds is more imho rather unrealistic ( we need to give real-time metrics) so I would expect 10 Seconds as maximal timeout period. So waiting 10 second as first start is not really an issue. E.g we have software that initialize with much more initial timeout and user are ok. 

A part of the IF and else cognitive load, there is also the wrong variable name `started` doesn't refer much to anything. ( this should improved).

I didn't catch this on the previous PR because was to much things changed. :grin: 

@ayoub-belarbi thoughts on this?

Reply:
the variable `started` if we decided to keep it, we should rename and add some comments to explain its mechanism. Started is to much ambigous

Reply:
@ashleyprimo  any pov here?

Reply:
ok fixed